Deployment/Release practices.md

```diff
@@ -121,16 +123,16 @@ Once you release your update into production, your app publishing status will be
 - **Removed** - The app is no longer available on Google Play or for existing users
 - **Update pending** - The update has been submitted and is being processed
 
-## Timed publishing
+## Managed publishing
 
-When you select timed publishing, your update needs to be processed before it can go live. Processing takes up to a few hours. When your update is processed, you'll see a **Go live** button. When you select it, you'll make the update available on Google Play within a few minutes.
+When you select managed publishing, your update needs to be processed before it can go live. Processing takes up to a few hours. When your update is processed, you'll see a **Go live** button. When you select it, you'll make the update available on Google Play within a few minutes.
 
-Unfortunately, Google Play still **doesn't offer automatic releases**. It is your responsibility to press the Go live button in order to push the update to the store. Additionally, note that timed publishing is **only available for updates!**
+Unfortunately, Google Play still **doesn't offer automatic releases**. It is your responsibility to press the Go live button in order to push the update to the store. Additionally, note that managed publishing is **only available for updates!**
 
-To turn on timed publishing:
+To turn on managed publishing:
 
-1. Go to the **Store presence** section in the sidebar
-2. At the bottom of your screen, go to the **Timed publishing** switch
-3. Review the information and press **Turn on**
+1. Go to the **Publishing overview** section in the sidebar
+2. At the right side of your screen, press the **Manage** button
+3. Review the information, select the switch **Managed publishing on ** and press the  **Save** button
```

Comment:
I think this is a small formatting issue with bold text, but it might be just a bug in this diff tool. I believe this should be like this **Managed publishing on** ... the second ** should not have a space prefixed